Program.cs

```diff
@@ -294,6 +308,22 @@ public void ShowNotification(string msg, ToolTipIcon toolTipIcon)
             notifyIcon.BalloonTipText = msg;
             notifyIcon.ShowBalloonTip(10);
         }
+
+        private void StartDaemon()
+        {
+            var process = new Process();
+            process.StartInfo.CreateNoWindow = true;
+            process.StartInfo.UseShellExecute = false;
+            process.StartInfo.FileName = string.Format("{0}\\{1}\\crc.exe",
+                Environment.GetFolderPath(Environment.SpecialFolder.ProgramFiles), @"CodeReady Containers");

```

Comment:
It implies to be installed with the MSI correct?

Reply:
Yes, correct, like the macOS tray. If running in debug build (from visual studio) it'll look for `crc` in `$GOBIN`. But the releases from github can't be used independently without some manual steps.